From aba3273273e826c6dc90f197ca9a3e800e826891 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Fri, 5 Nov 2021 12:41:35 -0400
Subject: [PATCH] ppp: fix build against 5.15 headers

The 5.15 kernel has removed ipx support, along with the userspace
visible header.

This support wasn't used previously (as it hasn't been very well
maintained in the kernel for several years), so we can simply
disable it in our build and wait for upstream to do a release that
drops the support.

Upstream-Status: Inappropriate [OE-specific configuration/headers]

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 pppd/Makefile.linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pppd/Makefile.linux b/pppd/Makefile.linux
index 22837c5..23b9b22 100644
--- a/pppd/Makefile.linux
+++ b/pppd/Makefile.linux
@@ -91,7 +91,7 @@ MAXOCTETS=y
 
 INCLUDE_DIRS= -I../include
 
-COMPILE_FLAGS= -DHAVE_PATHS_H -DIPX_CHANGE -DHAVE_MMAP -pipe
+COMPILE_FLAGS= -DHAVE_PATHS_H -DHAVE_MMAP -pipe
 
 CFLAGS= $(COPTS) $(COMPILE_FLAGS) $(INCLUDE_DIRS) '-DDESTDIR="@DESTDIR@"'
 
-- 
2.25.1

